src/arch/riscv/isa/formats/vector_arith.isa

```diff
@@ -858,7 +858,7 @@ def format VectorFloatNarrowingCvtFormat(code, category, *flags) {{
     set_src_reg_idx += setSrcWrapper(src3_reg_id)
     set_src_reg_idx += setSrcVm()
     code = maskCondWrapper(code)
-    code = eiDeclarePrefix(code)
+    code = eiDeclarePrefix(code, widening=True)
```

Comment:
This is a narrowing instruction so having the parameter "widening=True" is quite odd.

Reply:
I agree that it's a bit absurd.
I just followed the pattern that has been done before.
Perhaps we could introduce another parameter called `narrowing` to reduce confusion?